test/integration/test-amp-ad-3p.js

```diff
@@ -159,6 +159,10 @@ describes.realWin('3P Ad (with AmpContext experiment)', {
       });
     });
 
+    afterEach(() => {
+      toggleExperiment(fixture.win, '3p-use-ampcontext', false);
```

Comment:
Might be slightly cleaner to change the code in `beforeEach` to:
```
toggleExperiment(fixture.win, '3p-use-ampcontext', /* opt_on */ true,
    /* opt_transientExperiment */ true);
```
This way we don't assume that the default value for the experiment is "off" and the cookie setting is probably the issue. I tried testing this on saucelabs and didn't see the error.